### PR TITLE
fix: add build tag to disable dynamic link with libwasmer.so

### PR DESCRIPTION
**Pull request by antJack**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

**Review by taoyuanyuan — APPROVED** [review]
LGTM

File: pkg/wasm/runtime/wasmer/instance.go [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// +build Wasmer
```
Comment (zonghaishang) [inline pkg/wasm/runtime/wasmer/instance.go:1]: change Wasmer to wasmer

File: pkg/wasm/runtime/wasmer/types.go [hunk 2]
```diff
@@ -1,3 +1,5 @@
+// +build Wasmer
```
Comment (zonghaishang) [inline pkg/wasm/runtime/wasmer/types.go:1]: change Wasmer to wasmer

File: pkg/wasm/runtime/wasmer/module.go [hunk 3]
```diff
@@ -1,3 +1,5 @@
+// +build Wasmer
```
Comment (zonghaishang) [inline pkg/wasm/runtime/wasmer/module.go:1]: change Wasmer to wasmer

File: pkg/wasm/runtime/wasmer/vm.go [hunk 4]
```diff
@@ -1,3 +1,5 @@
+// +build Wasmer
```
Comment (zonghaishang) [inline pkg/wasm/runtime/wasmer/vm.go:1]: change Wasmer to wasmer

**Review by zonghaishang — CHANGES_REQUESTED** [review]

**Comment by zonghaishang** [comment]
https://dave.cheney.net/2013/10/12/how-to-use-conditional-compilation-with-the-go-build-tool

**Review by taoyuanyuan — APPROVED** [review]
LGTM

**Review by wangfakang — APPROVED** [review]
LGTM.

**Pull request merged**